File: 1528-KidsWithTheGreatestNumberOfCandies/1528-KidsWithTheGreatestNumberOfCandies.scala
// Last updated: 6/25/2026, 9:10:53 AM
object Solution {
    def kidsWithCandies(candies: Array[Int], extraCandies: Int): Array[Boolean] = {
        candies.map(x=> x*x).foreach(println)
        candies.map(_+extraCandies >= candies.max)
    }
}

File: 1528-KidsWithTheGreatestNumberOfCandies/1528-KidsWithTheGreatestNumberOfCandies.py
# Last updated: 6/25/2026, 9:10:49 AM
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """
        result=[False]*len(candies)
        m=max(candies)
        for i in range(len(candies)):
            if(candies[i]+extraCandies>=m):
                result[i]=True
        return result
            
